poc/prg.sage

```diff
@@ -138,10 +153,15 @@ class PrgFixedKeyAes128(Prg):
         ]
         return concat(hashed_blocks)[offset:offset+length]
 
-    # The multi-instance tweakable circular correlation-robust hash function of
-    # [GKWWY20] (Section 4.2). The tweak here is the key that stays constant for
-    # all PRG evaluations of the same client, but differs between clients.
     def hash_block(self, block):
+        """
+        The multi-instance tweakable circular correlation-robust hash
+        function of [GKWWY20] (Section 4.2). The tweak here is the key
+        that stays constant for all PRG evaluations of the same client,
+        but differs between clients.
+
+        Function `AES128(key, block)` is the AES-128 blockcipher.
```

Comment:
This second paragraph is copied from the corresponding comment in the Markdown file.